mimic/resource.py

```diff
@@ -1,10 +1,27 @@
 """
-Resources for Mimic's core.
+Resources for Mimic's core application.
 """
-
 import json
 
+from io import StringIO
+
+from six import text_type
+
 from twisted.web.resource import NoResource
+from twisted.web.server import Request, Site
+
+try:
```

Comment:
Could we please add a comment to explain this, so we know to get rid of it when we make decisions on versions supported and such.